src/Extend/Theme.php

```diff
@@ -52,6 +53,32 @@ public function overrideFileSource(string $file, string $newFilePath, string $ex
         return $this;
     }
 
+    /**
+     * This method allows you to add custom Less functions.
+     *
+     * **Example usage:**
+     * ```php
+     * use
+     *
+     * (new Extend\Theme)
+     *     ->addLessFunction('is_flarum', function (Less_Tree_Quoted $text) {
```

Comment:
emphasis on the naming of the function needing to be dashed case, I think that's the only way it worked locally for me.

Reply:
additionally, I would rather simplify the callback to only take a normal mixed type value and return any mixed type value, and we transform those values properly, to have a layer of abstraction between the less parser's implementation and the purpose of the extender.